package cal;

import java.io.IOException;

public class Calculator {
    int plus(int i , int j ) {
        return i+j;
    }

    int minus(int i , int j ) {
        return i-j;
    }

    public static void main(String[] args) throws Exception {
        try {
            func();
        } catch (IllegalArgumentException e) {

        }

    }

    static void func() throws Exception {
        Exception e = null;
        for (int i =0; i<4; i++) {
            boolean check =
        }
    }

}
